xbmc/cores/dvdplayer/DVDDemuxers/DVDDemuxFFmpeg.cpp

```diff
@@ -439,6 +439,19 @@ bool CDVDDemuxFFmpeg::Open(CDVDInputStream* pInput, bool streaminfo, bool filein
 
   if (m_streaminfo)
   {
+    if (CSettings::GetInstance().GetBool(CSettings::SETTING_AUDIOOUTPUT_SUPPORTSDTSHDCPUDECODING))
```

Comment:
You seem to know already that vero, Pi1 or ION-1 are too slow. Why this setting? IMO much better to disable it on those platforms automatically.